### PR TITLE
Fix Build Out Process for Relative rather then Absolute

### PR DESCRIPTION
**Pull request by sylus**
Fix Build Out Process for Relative rather then Absolute

### CODE REVIEW

**Comment by sylus** [comment]
Waiting on Travis to see if new build out works ^_^

File: build-wetkit.make [hunk 1]
```diff
@@ -1,10 +1,6 @@
...
 core = 7.x
 
-includes[] = https://raw.github.com/wet-boew/wet-boew-drupal/master/drupal-org-core.make
-
-; Add wetkit to the full Drupal distro build
-projects[wetkit][type] = profile
-projects[wetkit][download][type] = git
-projects[wetkit][download][url] = https://github.com/wet-boew/wet-boew-drupal.git
-projects[wetkit][download][revision] = master
+projects[drupal][version] = 7.15
```
Comment (patcon) [inline build-wetkit.make:4]: This line should probably be `drupal-org-core.make`. Only the top two lines need to be in each makefile

**Comment by patcon** [comment]
Can I ask what you're trying to accomplish here? I'm trying to figure out whether there might be a more intuitive way to do it. If it's that you're trying to avoid having drushmake reach out and fetch the install profile again (from HEAD of master, as specified in `build-wetkit.make`), then there are other ways that I feel are more foolproof.

You can `cat build-wetkit.make` to stdout, manipulate the ouput with `sed` to replace a value in a makefile (like a branch name) with the specific reference you want (like the current branch name, or even a commit hash), and then run drush out that output instead of the file:
https://github.com/myplanetdigital/secondlevel-rerun-modules/blob/develop/2ndlevel/commands/build/default.sh#L37

That way, you can known that even when drush does the thing it wants to and reaches out into the internet again, it will only fetch the version consistent with what it originally cloned

**Comment by sylus** [comment]
Hey @patcon I am thinking I can do the initial push and we can comment on improvements in an issue and issue another pull request?

**Pull request merged**